.github/CODEOWNERS

```diff
@@ -26,8 +26,10 @@
 /core/templates/services/contextual/logger.service.ts @bansalnitish @srijanreddy98
 /core/templates/services/contextual/logger.service.spec.ts @bansalnitish @srijanreddy98
 /core/templates/services/guppy-configuration.service.ts @bansalnitish @srijanreddy98
+/core/templates/services/guppy-initialization.service.ts @bansalnitish @srijanreddy98
```

Comment:
Why it is adding me and @srijanreddy98 as codeowner?

Reply:
@bansalnitish @srijanreddy98 
I saw that the other services have both of you as code-owners which is why I thought all services should be owned by you guys. Should I assign someone else instead?

Reply:
Maybe move this and the above line to the "Interaction" section?

Reply:
Done.